src/main/resources/assets/mythicmetals/lang/zh_cn.json

```diff
@@ -1,727 +1,1201 @@
+	"text.config.mythicmetals-config.option.netherBanglum": "生成下界聚爆石矿石",
+	"text.config.mythicmetals-config.option.orichalcum": "生成山铜矿石",
+	"text.config.mythicmetals-config.option.osmium": "生成锇矿石",
+	"text.config.mythicmetals-config.option.palladium": "生成钯金矿石",
+	"text.config.mythicmetals-config.option.platinum": "生成铂金矿石",
+	"text.config.mythicmetals-config.option.prometheum": "生成钷矿石",
+	"text.config.mythicmetals-config.option.quadrillum": "生成兆金矿石",
+	"text.config.mythicmetals-config.option.runite": "生成符石矿石",
+	"text.config.mythicmetals-config.option.shieldPosition": "点金石盾牌HUD位置",
+	"text.config.mythicmetals-config.option.silver": "生成银矿石",
+	"text.config.mythicmetals-config.option.starrite": "生成星辰矿石",
+	"text.config.mythicmetals-config.option.stormyx": "生成飓霆矿石",
+	"text.config.mythicmetals-config.option.tin": "生成锡矿石",
+	"text.config.mythicmetals-config.option.unobtainium": "生成叵得矿石",
+	"text.config.mythicmetals-config.option.unobtainiumTemplateChance": "叵得合金模板概率",
+	"text.config.mythicmetals-config.option.unobtainiumTemplateChance.tooltip": "生成于远古城市的宝箱中",
+	"text.config.mythicmetals-config.title": "神话金属配置",
+
+	"tooltip.carmot_staff.beacon": "独特能力 - 范围增益",
+	"tooltip.carmot_staff.bronze": "独特能力 - 雷霆打击",
+	"tooltip.carmot_staff.carmot": "独特能力 - 范围治疗",
+	"tooltip.carmot_staff.command_block": "创造之力 - 右击升天",
+	"tooltip.carmot_staff.copper": "独特能力 - 避雷针",
+	"tooltip.carmot_staff.empty_staff": "用方块在法杖图标上右击以插入",
+	"tooltip.carmot_staff.gold": "独特能力 - 幸运 I",
+	"tooltip.carmot_staff.iron": "独特能力 - 投掷敌人",
+	"tooltip.carmot_staff.locked": "已锁定",
+	"tooltip.carmot_staff.midas_gold": "独特能力 - 范围幸运 II",
+	"tooltip.carmot_staff.note_block": "独具一格的乐器——快试试吧！",
+	"tooltip.carmot_staff.sponge": "独特能力 - 吸水",
+	"tooltip.carmot_staff.stormyx": "独特能力 - 弹射物屏障",
+	"tooltip.carmot_staff.unique": "独特",
+	"tooltip.carmot_staff.unobtainium": "ONE PIECE是真实存在的！",
+	"tooltip.midas_gold.fold_counter": "折叠锻打次数：%s",
+	"tooltip.midas_gold.level.0": "可以在锻造台中使用迈达斯金块折叠锻打",
+	"tooltip.midas_gold.level.1": "你需要足量迈达斯金才能解锁其真正的力量……",
+	"tooltip.midas_gold.level.2": "超过一百块了。令人印象深刻！继续加油！",
+	"tooltip.midas_gold.level.3": "你为什么要造出这么多块？只需要普通金……",
+	"tooltip.midas_gold.level.5": "迈达斯国王会以此为傲",
+	"tooltip.midas_gold.level.6": "迈达斯国王会以此为傲",
+	"tooltip.midas_gold.level.7": "迈达斯国王会以此为傲",
+	"tooltip.midas_gold.level.8": "迈达斯国王会以此为傲",
+	"tooltip.midas_gold.level.9": "迈达斯国王会以此为傲",
+	"tooltip.midas_gold.level.10": "一把出色的武器！也许它就是你走得更远的关键……",
+	"tooltip.midas_gold.level.11": "供剑使用的皇家升级",
+	"tooltip.midas_gold.level.12": "如果最初，你没有成功……",
+	"tooltip.midas_gold.level.13": "……试一下，再试一下。这样你就会充满勇气。",
+	"tooltip.midas_gold.level.14": "皇家迈达斯金剑的真正实力只会在那个时候展现……",
+	"tooltip.midas_gold.level.15": "……一旦你到达了旅途的终点。",
+	"tooltip.midas_gold.level.16": "经过千百次折叠锻打，剑刃变得纯净。",
+	"tooltip.midas_gold.level.17": "你击败的任何事物都会给你更多你真正渴望的东西：",
+	"tooltip.midas_gold.level.18": "更多的迈达斯金！",
+	"tooltip.midas_gold.level.19": "最后一点了，你可以的！",
+	"tooltip.midas_gold.level.20": "你的剑中已蕴含了迈达斯国王的无上力量",
+	"tooltip.midas_gold.level.21": "chyz在这里",

```

Comment:
This is a joke line, and can be replaced with something similar. An example is in the German PR: https://github.com/Noaaan/MythicMetals/pull/187/files#diff-b810846f6a469a7519ba3d121dc5d0bb1fcbd8dfd61d4f8de5e2bd46e1b751f4R731

Reply:
ok,I will update missing translation key and this key later